### PR TITLE
Cleanup of plugins in deprecated format

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Cleanup of plugins in deprecated format.
This pr is not a part of "End Gama" and can be merged after GA.
Added redirection `Redirect temp /plugins /v3/plugins` that would be eventually replaced  with
`Redirect permanent  /v3/plugins /plugins`

### CODE REVIEW

**Comment by skabashnyuk** [comment]
CC @ibuziuk

File: Dockerfile [hunk 1]
```diff
@@ -9,27 +9,17 @@
 FROM mikefarah/yq as builder
 RUN apk add --no-cache bash
 COPY .htaccess README.md ./scripts/*.sh /build/
-COPY /plugins /build/plugins
-COPY /v2 /build/v2
 COPY /v3 /build/v3
 WORKDIR /build/
-RUN ./check_plugins_location_v1.sh
-RUN ./check_plugins_location_v2.sh v2
 RUN ./check_plugins_location_v2.sh v3
```
Comment (amisevsk) [inline Dockerfile]: Should this script be renamed to just `check_plugins_location.sh`?

File: Dockerfile [hunk 2]
```diff
@@ -9,27 +9,17 @@
...
 RUN ./check_plugins_images.sh
 RUN ./set_plugin_dates.sh
-RUN ./check_plugins_viewer_mandatory_fields_v1.sh
-RUN ./check_plugins_viewer_mandatory_fields_v2.sh
 RUN ./check_plugins_viewer_mandatory_fields_v3.sh
 RUN ./ensure_latest_exists.sh
-RUN ./index.sh > /build/plugins/index.json
-RUN ./index_v2.sh v2 > /build/v2/plugins/index.json
 RUN ./index_v2.sh v3 > /build/v3/plugins/index.json
```
Comment (amisevsk) [inline Dockerfile]: We could rename this to `index.sh` or `index_v3.sh` for consistency (since `v2` is gone as of this PR).

**Comment by skabashnyuk** [comment]
@l0rd @rhopp what is your opinion about this pr? Should we postpone it for 7.1 to make things safer?

**Comment by l0rd** [comment]
@skabashnyuk I would do that now

**Pull request merged**